auto_nag/user_activity.py

```diff
@@ -69,12 +80,22 @@ def handler(user, data):
                 data.add(user["name"])
 
         data = set()
-        BugzillaUser(
-            user_data=data,
-            user_names=set(user_emails),
-            user_handler=handler,
-            include_fields=["name", "can_login"],
-        ).wait()
+        user_emails = (
+            user_emails if isinstance(user_emails, list) else list(user_emails)
+        )
+
+        step = 200
+        start = 0
+        end = start + step
+        while start < len(user_emails):
+            chunk = user_emails[start:end]
+            BugzillaUser(
+                user_data=data,
+                user_names=chunk,
+                user_handler=handler,
+                include_fields=["name", "can_login"],
+            ).wait()
+            start, end = end, end + step
```

Comment:
Maybe the chunking should be implemented in libmozdata so it is transparent to users of the library (just like for bugs).

Reply:
Done in 8039836bc407ffac4ed92a70c3ce3926895f37b1